app/src/main/java/com/vincentengelsoftware/androidimagecompare/helper/SlideHelper.java

```diff
@@ -85,6 +95,7 @@ public void onProgressChanged(SeekBar seekBar, int i, boolean b) {
                         )
                 );
 
+                hideShow.setImageResource(R.drawable.ic_unhide_vector);
```

Comment:
Please add `hideShow.setImageResource(R.drawable.ic_hide_vector);` between line 75 and 76. So that the hide button correctly switches, when the front images is hidden.

Reply:
Oh and also line 85-86

Reply:
Thanks, good point!
Should it be done in TransparentHelper too? And which visual feedback will the user have there when clicking on hideShow to reset visibility?

Reply:
Yes add it to TransparentHelper too 👍 

I assume you mean feedback for when the transparency is set to 0 and the user clicks on the hideShow button? I am not sure about a visual feedback for that case. But I think the users will figure it out ;)

If you have an idea that would be great, but if not I think we can just leave that case out.

Reply:
Hide Button now updates correctly.
As `return` is fired when seekBar progress exceeds the limits, a subsequent click on hide will reveal the last state within the limits and update the seekBar accordingly.

Reply:
Not sure what `width == 0` in line 82 is for. So far I ignored this case in `OverlaySlideActivity` line 60.

Reply:
It is a fallback for when the image is small.

With `bitmapSource.getWidth() <= 49` and `progress = 2`

`cutFromRightToLeft.value && ((progress <= 1)` => `false`

but

`(49 * 2) / 100` => `0`

Now that I check it again, I should take a look at the beginning of the method, because it should be checked there as well.

Reply:
Ah ok, thanks for explanation